README.md

```diff
@@ -86,16 +93,16 @@ Databox has a number of platform components, divided into two parts:  Core and U
 * [driver-phillips-hue](https://github.com/me-box/driver-phillips-hue)
 * [driver-os-monitor](https://github.com/me-box/driver-os-monitor)
 * [driver-twitter](https://github.com/me-box/driver-twitter)
-* [driver-tplink-smart-plug](https://github.com/me-box/driver-tplink-smart-plug)    
-#### Apps 
+* [driver-tplink-smart-plug](https://github.com/me-box/driver-tplink-smart-plug)
+#### Apps
 * [app-light-graph](https://github.com/me-box/app-light-graph)
 * [app-twitter-sentiment](https://github.com/me-box/app-twitter-sentiment)
 * [app-os-monitor](https://github.com/me-box/app-os-monitor)
-                
+
 ### Libraries for writing drivers and apps
 For writing a new driver or app for Databox, one needs [Databox Api's](./documents/api_specification.md). To make app/driver development easy, we have wrapped Databox api's in [nodejs](https://nodejs.org/en/), [python](https://docs.python.org/3.4/library/index.html) and [go](https://golang.org/). Using any of these libraries, a developer can build their databox app/driver.
```

Comment:
s/[aA]pi/API/g

Reply:
I didn't get this comment.

Reply:
(Regex for replace instances of Api and api with API globally)

Reply:
Got it :)

Reply:
I've fixed this one